dvc/repo/data.py

```diff
@@ -10,124 +9,98 @@
     Iterator,
     List,
     Optional,
-    Tuple,
     TypedDict,
     Union,
 )
 
-from dvc.fs.git import GitFileSystem
-from dvc.ui import ui
-
 if TYPE_CHECKING:
     from dvc.repo import Repo
     from dvc.scm import Git, NoSCM
-    from dvc_data.hashfile.db import HashFileDB
-    from dvc_data.hashfile.hash_info import HashInfo
-    from dvc_data.hashfile.obj import HashFile
+    from dvc_data.index import DataIndex
 
 
 def posixpath_to_os_path(path: str) -> str:
     return path.replace(posixpath.sep, os.path.sep)
 
 
-def _in_cache(obj: "HashInfo", cache: "HashFileDB") -> bool:
-    from dvc_objects.errors import ObjectFormatError
+def _adapt_typ(typ):
```

Comment:
This probably should just be a dictionary instead of a function.

```python
state_map = {
    MODIFY: "modified",
    ADD: "added",
    DELETE: "deleted",
    UNCHANGED: UNCHANGED,
    UNKNOWN: UNKNOWN,
}
```